<launch>

<arg name="fcu_url" default= "udp://:14540@127.0.0.1:14557"/>
<arg name="gcs_url" default="" />
<arg name="tgt_system" default="1" />
<arg name="tgt_component" default="1" />



<node name="mavros_earth" pkg="mavros" type="mavros_node" output="screen">
	<!--param name="fcu_url" value="/dev/ttyUSB0:57600"-->
 	<param name="fcu_url" value="udp://:14540@127.0.0.1:14557"/>
	<param name="gcs_url" value="$(arg gcs_url)" />
	<param name="target_system_id" value="1" />
	<param name="target_component_id" value="1" />
</node>

<node name="start_earth_node" pkg="solar_system_formation" type="start_earth_node" output="screen">
</node>






<node name="mavros_moon" pkg="mavros" type="mavros_node" output="screen">
	<param name="fcu_url" value="/dev/ttyUSB0:57600"/>
	<!--param name="fcu_url" value="udp://:14540@127.0.0.1:14557"-->
	<param name="gcs_url" value="$(arg gcs_url)" />
	<param name="target_system_id" value="3" />
	<param name="target_component_id" value="1" />
</node>

<node name="start_moon_node" pkg="solar_system_formation" type="start_moon_node" output="screen">
</node>




</launch>

